src/libsystemd/sd-netlink/netlink-socket.c

```diff
@@ -238,6 +238,32 @@ int socket_write_message(sd_netlink *nl, sd_netlink_message *m) {
         return k;
 }
 
+int socket_writev_message(sd_netlink *nl, sd_netlink_message *m[], unsigned int msgcount) {
```

Comment:
`msgcount` is ultimately the size of an object in memory hence please use `size_t` type